src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -786,6 +792,15 @@ void ConnectedSubchannel::Ping(grpc_closure* on_initiate,
   elem->filter->start_transport_op(elem, op);
 }
 
+void ConnectedSubchannel::DisconnectDueToFork() {
+  grpc_transport_op* op = grpc_make_transport_op(nullptr);
+  grpc_channel_element* elem;
+  op->disconnect_with_error =
+      GRPC_ERROR_CREATE_FROM_STATIC_STRING("Disconnected due to fork");
```

Comment:
It might be good to explicitly set the status on this error to ensure that calls fail with the right status code (probably INTERNAL_ERROR).